pkg/api/unversioned/types.go

```diff
@@ -300,6 +300,10 @@ type APIVersions struct {
 	TypeMeta `json:",inline"`
 	// versions are the api versions that are available.
 	Versions []string `json:"versions"`
+	// a map of CIDR to address of the server that is serving this group. Clients can use the appropriate address as per the CIDR that they match.
```

Comment:
Is this CIDR for client IP or a destination IP?  The description is not clear.  Is it the client that does the mapping or do you map it server-side?